### DISCUSSION TITLE
Cloud Solutioning for Detecting Features of ears in a elephant.

### DISCUSSION THREAD

**Discussion by Tejas-ChandraShekarRaju** [cia-labs/vision-tech#24] [Ideas]
@johnkaramchand  @suryamurugan  @Thanmay47  Use this space to share and discuss ideas. Someone can start by updating today's MoM

**Comment by johnkaramchand** [comment]
###MoM for the ISM tech review meet

- We are in consensus that the overarching process/plan for modelling is to
    - Have a mixture of models for different tasks
    - Each of these need annotated data for training
    - Tasks are either object identification or classifying against the 100+ features that Feral tracks
- We plan to do a trial of the process on the cloud, with data scrapped out of the internet. This will help us validate the ML process without being limited by both compute and the inability to use off the shelf / cloud native solutions. Once we have relative confidence on the process, we will solution for Feral's inability to use the public cloud. 
- We discussed and agreed that we need to complete writing the paper describing our efforts, so that we can share it for feedback. 
- We discussed the need to have ML scientist to drive the research process. Also, potentially using them to write the paper / plan too. 
- We will meet in a week's time to wrap up the paper and kickstart the process of sharing it around for feedback. 
- We currently also plan to hire interns for ML research. In our context, the scope of research is to try out various solutions and models against our use case and compare and analyze these results. 


kindly add any other point I might have missed

**Reply by Tejas-ChandraShekarRaju** [reply]
Hugging face seem to have good vision models. Need to check that out. To start with,  I'll checkout GCP solutions. 
CC : @johnkaramchand  @suryamurugan  @Thanmay47
